File: Java Automation QA Sample/src/main/java/in/wrapstore/automation/locators/StaticLocators.java
package in.wrapstore.automation.locators;

import org.openqa.selenium.By;

public class StaticLocators {

    // -------------------------------
    // Home Page
    // -------------------------------
    public static final By STORE_LINK = By.linkText("Store");

    // -------------------------------
    // Store Page
    // -------------------------------
    public static final By CATEGORY_ARMOUR_GLASS = By.xpath("//img[@alt='Armour Glass Case']");

    // -------------------------------
    // Category Page
    // -------------------------------
    public static final By PRODUCT_BATMAN_TOON = By.xpath("//h2[contains(text(),'Batman Toon Glass Case')]");

    // -------------------------------
    // Product Page
    // -------------------------------
    public static final By DROPDOWN_BRAND = By.id("wrapstore_brand");
    public static final By DROPDOWN_MODEL = By.id("wrapstore_device");
    public static final By ADD_TO_CART_BTN = By.cssSelector("button.single_add_to_cart_button[name='add-to-cart']");


    // -------------------------------
    // Cart Page
    // -------------------------------
    public static final By CART_TITLE = By.cssSelector("h1.entry-title");
}

File: Python Automation QA Sample/locators/static_locators.py
# in/wrapstore/automation/locators/static_locators.py

from selenium.webdriver.common.by import By


class StaticLocators:
    # -------------------------------
    # Home Page
    # -------------------------------
    STORE_LINK = (By.LINK_TEXT, "Store")

    # -------------------------------
    # Store Page
    # -------------------------------
    CATEGORY_ARMOUR_GLASS = (By.XPATH, "//img[@alt='Armour Glass Case']")

    # -------------------------------
    # Category Page
    # -------------------------------
    PRODUCT_BATMAN_TOON = (By.XPATH, "//h2[contains(text(),'Batman Toon Glass Case')]")

    # -------------------------------
    # Product Page
    # -------------------------------
    DROPDOWN_BRAND = (By.ID, "wrapstore_brand")
    DROPDOWN_MODEL = (By.ID, "wrapstore_device")
    ADD_TO_CART_BTN = (By.CSS_SELECTOR, "button.single_add_to_cart_button[name='add-to-cart']")

    # -------------------------------
    # Cart Page
    # -------------------------------
    CART_TITLE = (By.CSS_SELECTOR, "h1.entry-title")
